packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -535,90 +542,132 @@ export class JobService {
     }
   }
 
-  public async cancelCronJob() {
+  public async refundCronJob() {
     // TODO: Add retry policy and process failure requests https://github.com/humanprotocol/human-protocol/issues/334
     const jobEntity = await this.jobRepository.findOne(
       {
-        status: JobStatus.TO_CANCEL,
+        status: JobStatus.TO_REFUND,
         retriesCount: LessThanOrEqual(JOB_RETRIES_COUNT_THRESHOLD),
         waitUntil: LessThanOrEqual(new Date()),
+        updatedAt: LessThanOrEqual(_5_MINS)
```

Comment:
why you add this condition?

Reply:
@portuu3

Reply:
This period ensures that the blockchain has successfully processed the cancellation request. It verifies that the escrow was canceled and confirms the funds were transferred once the job was initiated.